app/models/user.rb

```diff
@@ -71,6 +71,13 @@ def set_next_email_as_primary
     default_email_record.update_attributes(primary: true)
   end
 
+  # Loads unread popups of a user.
+  #
+  # @return [Array<UserNotification>] The unread popup notifications
+  def unread_popups
+    notifications.popups.unread_by(self).order(created_at: :desc)
```

Comment:
This part is the unread scope problem that we have not solved. Also I've attached the link in slack. The link is about some discussions with the owner of 'unread' gem.